profiles/commands.py

```diff
@@ -13,7 +14,7 @@ def update_profile_from_orcid_record(profile: Profile, orcid_record: dict) -> No
 
 
 def extract_email_addresses(orcid_record: dict, only_verified: bool = True) -> List[dict]:
-    orcid_emails = orcid_record.get('person', {}).get('emails', {}).get('email', {})
+    orcid_emails = jmespath.search('person.emails.email[*]', orcid_record) or []
```

Comment:
Seems to return `None` rather than an empty list.

Reply:
`family_name = jmespath.search('person.name."family-name".value', orcid_record) or []` returns `[]` when tested so it may be something to do with the `email[*]` part of the expression. Would expect the right side value to be assigned if `.search()` returned falsely.

Reply:
Reading the JMESPath spec think it will always be `None` if not set at all, so this will have to do.

Reply:
I can't seem to get this 'not' to return `[]` with some bits I have thrown at it. What is the data fixture you are testing with?